sql/explain.go

```diff
@@ -48,9 +51,20 @@ func (p *planner) Explain(n *parser.Explain) (planNode, error) {
 	if err != nil {
 		return nil, err
 	}
-	plan, err = markExplain(plan, mode)
-	if err != nil {
-		return nil, fmt.Errorf("%v: %s", err, n)
+	switch mode {
+	case explainDebug:
+		plan, err = markExplain(plan, mode)
```

Comment:
I'd name this one `markDebug` since it really behaves more like a normal select with the debug option, and I'd change `explainNode` -> `populateExplain` and possibly make it a method on `valuesNode`